File: csv/java/update/csv_update.java
// -----------------------------------------------------------------------
/*
	csv_update.java

					Jul/11/2011
*/
// -----------------------------------------------------------------------
import	java.util.HashMap;

// -----------------------------------------------------------------------
public class csv_update
{

// -----------------------------------------------------------------------
public	static void main (String [] args)
{
	System.out.println ("*** 開始 *** update ***");

	String text_file = args[0];

	String	id = args[1];
	int	population = Integer.parseInt (args[2]);
	System.out.print ("\tid = " + id);
	System.out.println ("\tpopulation = " + population);

	HashMap <String, HashMap<String,String>> dict_aa
		= text_manipulate.csv_read_proc (text_file);

	text_manipulate.dict_update_proc (dict_aa,id,population);

	text_manipulate.dict_display_proc (dict_aa);

	text_manipulate.csv_write_proc (text_file,dict_aa);

	System.out.println ("*** 終了 ***");
}

// -----------------------------------------------------------------------
}
// -----------------------------------------------------------------------

File: csv/clojure/update/csv_update.clj
; -----------------------------------------------------------------
;
;	csv_update.clj
;
;					Oct/27/2011
;
; -----------------------------------------------------------------
(load-file "/var/www/data_base/common/clojure_common/text_manipulate.clj")
; -----------------------------------------------------------------
(println "*** 開始 ***")
(def file_text (first *command-line-args*))
(def id_in (second *command-line-args*))
(def population_in (second (rest *command-line-args*)))
(def key_in (keyword id_in))
(println file_text)
(print id_in)
(print "\t")
(println population_in)
;
(def dict_aa (csv_read_proc file_text))

(def dict_aa (assoc dict_aa key_in (new_unit_gen_proc dict_aa key_in population_in)))

(csv_write_proc file_text dict_aa)
(dict_display_proc dict_aa)
;
(println "*** 終了 ***")
; -----------------------------------------------------------------
